src/util/helpers.tsx

```diff
@@ -55,6 +55,10 @@ export const getBrowserFormatDate = (d: Date): string =>
     d.getHours(),
   )}:${pad(d.getMinutes())}`;
 
+export const nonBreakingSpaces = (text: string): string => {
+  return text.replace(/ /g, "\u00A0");
```

Comment:
I wonder if it would make sense to use `\s` to match any whitespace character, but this would include new lines, so maybe safer without it.